.github/workflows/update-testing-files.yml

```diff
@@ -25,6 +25,10 @@ jobs:
 
       - name: Install DANDI
         run: pip install -e .[dandi]
+      - name: Install older PyNWB version to generate example invalid files
+        run: |
+          pip uninstall pynwb
+          pip install pynwb==2.1.0
```

Comment:
(the workflow will continue failing on this PR since the deployment is pinned to the version on `main` - I could pin it to this branch but then I'd have to go back and fix it after merge so it's w/e you prefer)